pennylane/optimize/qng.py

```diff
@@ -223,7 +223,9 @@ def step_and_cost(
         # unwrap from list if one argument, cleaner return
         if len(new_args) == 1:
             return new_args[0], forward
-        return new_args, forward
+        # The following is essentially not implemented yet, as only
+        # a single metric tensor can be processed.
+        return new_args, forward  # pragma: no cover
```

Comment:
I would suggest _not_ putting the no-cover here, as we should be aware that this is not covered in our coverage checks.

Reply:
I would also extend this comment to explain why this isn't covered/what has to be done to support it :)

Reply:
Does this require a modification of coverage CI checks?

Reply:
Would you be fine with a `TODO:` comment to mark it as an open task?

Reply:
Yep a TODO is fine!

> Does this require a modification of coverage CI checks?

I can override codecov here :)

Reply:
Okay! :) With the TODO in place I would leave the no-cover in. Or do you prefer not to put it? :)

Reply:
I would prefer to not include it :)